src/lldpd/.gitignore

```diff
@@ -1,4 +1,3 @@
-*
```

Comment:
@ZhaohuiS what's the reason to change to change the .gitignore file?

Reply:
@StormLiangMS This line blocked new patch file under patch. It was ignored.
```
$ git check-ignore -v 0012-fix-recreate-socket-on-ifindex-change.patch      
src/lldpd/.gitignore:1:*        0012-fix-recreate-socket-on-ifindex-change.patch
```